### PR TITLE
Group 2

### PR DESCRIPTION
**Pull request by PtrBld**

### CODE REVIEW

File: library.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
+    def print_longest_crime_audio_book(self) -> str:
+        crime_audio_books: list[Book] = []
```
Comment (janavatter) [inline library.py:10]: this variable is never used again

File: library.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        max: Optional[Book] = None
+
+        # print banner
+        print("*********************************************")
+        print("********* Longest Crime Audio Book **********")
+        print("*********************************************")
+
+        for b in self.get_books():
+            if Genre.CRIME in b.genres:
```
Comment (janavatter) [inline library.py:19]: merge in one line of code "if ... in .. and .. "

File: library.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                if b._book_type == "Audio":
+                    if max is not None and max.duration < b.duration:
+                        max = b
+                    # end if
+                # end if
+            else:
```
Comment (janavatter) [inline library.py:25]: not necessary

File: library.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                pass
+            # end else
+        # end for
+
+        print(f"Title: {max.title}")
+        print(f"Author: {max.authors}")
+        print(f"Duration: {max.duration}")
+        print(f"______________________________")
+        return str(max)
```
Comment (janavaa) [inline library.py:34]: maybe return integer instead of string

File: library.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+    # Constructor
```
Comment (janavatter) [inline library.py:36]: constructor usually is at the beginning of the class

File: library.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    def __init__(self, books: list[Book]):
+        self._books = books
+
+    _books: list[Book] = []
```
Comment (janavatter) [inline library.py:40]: we already have the class variable _books

File: library.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
```
Comment (janavatter) [inline library.py:8]: comment doesn't match the function, the function prints the longest crime audio book

**Review by janavatter — COMMENTED** [review]
we think there are some things that could be improved, please take a look at our comments

**Pull request closed without merging**